native/packages/react-native-bpk-component-chip/index.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import BpkChip from './src/BpkChip';
+import BpkDismissibleChip from './src/BpkDismissibleChip';
+
+export default BpkChip;
+export { BpkDismissibleChip };
```

Comment:
Do we need to export the type for the props?

Reply:
I'm not sure where that convention came from 🤔 is that something we always do? My understanding was that we exported things like `buttonTypes`, as I can see when consumers might need those, but I'm not sure when they'd need these.

Reply:
It's easy for me to do so I'm happy to do it, I just wanna understand what our convention is.

Reply:
that's why I've asked, not 100% sure 😅